var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -309,10 +320,9 @@ def make_defs(self):
         # When mixing compilers make sure that
         # $SPACK_ROOT/lib/spack/env/<compiler> have symlinks with reasonable
         # names and hack them inside lib/spack/spack/compilers/<compiler>.py
-        make_defs = [
-            "CC={0}".format(spack_cc),
-            "FC={0}".format(spack_fc),
-        ]
+        make_defs = ["CC={0}".format(spack_cc)]
+        if "~fortran" not in self.spec:
```

Comment:
Oh, this is a double negation because the variant is conditional? Maybe add a comment so that people don't change it into `+fortran in self.spec` later on?

Reply:
Oh, nothing special in mind. Let me simplify:
```suggestion
        if "+fortran" in self.spec:
```